ci/scripts/run-e2e-test.sh

```diff
@@ -70,6 +70,35 @@ sqllogictest -p 4566 -d dev -e postgres-extended './e2e_test/extended_query/**/*
 echo "--- Kill cluster"
 cargo make ci-kill
 
+if [[ "$RUN_COMPACTION" -eq "1" ]]; then
+    echo "--- e2e, ci-compaction-test, nexmark_q7"
+    cargo make clean-data
+    cargo make ci-start ci-compaction-test
+    # Please make sure the regression is expected before increasing the timeout.
+    sqllogictest -p 4566 -d dev './e2e_test/compaction/ingest_rows.slt'
+
+    # We should ingest about 100 version deltas before the test
+    echo "--- Wait for data ingestion"
+    # Poll the current version id until we have around 100 version deltas
+    delta_log_cnt=0
+    while [ $delta_log_cnt -le 95 ]
+    do
+        delta_log_cnt="$(./risedev ctl hummock list-version | grep -w '^ *id:' | grep -o '[0-9]\+' | head -n 1)"
+        echo "Current version $delta_log_cnt"
+        sleep 5
+    done
+
+    echo "--- Pause source and disable commit new epochs"
+    ./risedev ctl meta pause
+    ./risedev ctl hummock disable-commit-epoch
+
+    echo "--- Start to run compaction test"
+    cargo run -r --bin compaction-test -- --ci-mode true --state-store hummock+minio://hummockadmin:hummockadmin@127.0.0.1:9301/hummock001
```

Comment:
Can we also verify that we have indeed done enough checks, e.g via "Replay finished. Check times: {}, Uncheck times: {}". If "Check times" is 0, there must be something wrong but the test doesn't fail as expected.

Reply:
Good catch.